apps/acert.c

```diff
@@ -0,0 +1,840 @@
+/*
```

Comment:
This doesn't seem to be linked into the openssl command line app.
Likewise, the docs seem absent.
Tests are also missing.